feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-multi-selection-base.component.ts

```diff
@@ -35,11 +34,10 @@ export abstract class ConfiguratorAttributeMultiSelectionBaseComponent extends C
     this.attribute = attributeComponentContext.attribute;
     this.ownerKey = attributeComponentContext.owner.key;
     this.expMode = attributeComponentContext.expMode;
-    this.initDeltaRendering(
-      attributeComponentContext.isDeltaRendering,
+    this.initPriceChangedEvent(
+      attributeComponentContext.isPricingAsync,
       attributeComponentContext.attribute.key
```

Comment:
Do we _really_ have to call `this.initPriceChangedEvent()` it in every child class?

Can't we `inject(ConfiguratorAttributeCompositionContext)` in the super class `ConfiguratorAttributeBaseComponent`? and call `this.initPriceChangedEvent()` there? - so it will be self-sufficient?

Thanks to this, the child classes would be cleaner, wouldn't they?

Reply:
`inject(ConfiguratorAttributeCompositionContext)`  is only injected for components that can occur when configuring with CPS, but not to CPQ exclusive components and this was already done/decided before this PR, so i would like to avoid to refactor this now.